### PR TITLE
Stub sdkversion and cpumode sysctls

### PR DESCRIPTION
**Pull request by mikusp**

### CODE REVIEW

File: src/kernel/src/sysctl/mod.rs [hunk 1]
```diff
@@ -787,6 +811,37 @@ static KERN_ARANDOM: Oid = Oid {
...
+    link: Some(&KERN_CPUMODE), // TODO: Use a proper value.
+    number: Sysctl::KERN_SDKVERSION,
+    kind: Sysctl::CTLFLAG_RD
+        | Sysctl::CTLFLAG_MPSAFE
+        | Sysctl::CTLFLAG_CAPRD
+        | Sysctl::CTLTYPE_UINT,
+    arg1: None,
+    arg2: 0x09008031, // TODO: check what that means
+    name: "sdk_version",
+    handler: Some(Sysctl::kern_sdkversion),
```
Comment (mikusp) [inline src/kernel/src/sysctl/mod.rs:825]: This handler might be one of the generic ones, please suggest which one if you have it reversed

Comment (SuchAFuriousDeath) [reply src/kernel/src/sysctl/mod.rs:825]: I think it's sysctl_handle_int

Comment (mikusp) [reply src/kernel/src/sysctl/mod.rs]: Changed that

**Review by SuchAFuriousDeath — APPROVED** [review]
Good job

**Pull request merged**